docs/tryout.md

```diff
@@ -0,0 +1,390 @@
+    
+---
+    
+## Configure service provider to act as the client
+    
+You need to register your application as a service provider in WSO2 Identity Server.
+
+1. Log in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) using administrator credentials (`admin:admin`).
+
+2. Navigate to **Main** > **Identity** > **Service Providers** > **Add**.
+
+3. Enter a **Service Provider Name**. Optionally, enter a **Description**.
+    
+4. Click **Register**.    
+
+### Basic OAuth/OpenID Connect configuration
+
+Make the following changes to the created service provider.
+
+1. Expand **Inbound Authentication Configuration** > **OAuth/OpenID Connect Configuration** and click **Configure**.
+
+2. Enter the **Callback Url**. 
+
+3. Click **Add**. Note the **OAuth Client Key** and **OAuth Client Secret** that appear. 
+
+### Claim configuration
+
+Update claims for service provider:
+
+1. Expand **Claim Configuration** and select `http://wso2.org/claims/username` as the **Subject Claim URI**.
+
+
+   ![claim-config-username](img/claim-config-username.png)
+   
+
+2. Click **Update** to save the changes.
+
+---
+
+## Obtain the Protection API Access token (PAT)
+
+-   Execute the following curl command to obtain the PAT:
+    -   Be sure to replace the `<CLIENT_ID>` and `<CLIENT_SECRET>` tags with the values you obtained when you [configured the service provider for the resource server](#configure-service-provider-to-act-as-the-resource-server).
+    -   In this guide, the grant type that is used to obtain the PAT is the password grant type. Therefore, you need to pass the resource owners credentials in the curl command. 
+
+    **Request Format**
+    ```
+    curl -u <CLIENT_ID>:<CLIENT_SECRET> -k -d "grant_type=password&username=<USERNAME>&password=<PASSWORD>&scope=uma_protection internal_application_mgt_view" -H "Content-Type:application/x-www-form-urlencoded" https://<IS_HOST>:<IS_PORT>/oauth2/token
+
+    ```
+    **Sample Request**
+    ```curl
+    curl -u hwbR3jd2fikSApLKfv_wiwRWNSwa:q7Kb74s5dcK3FBh1MUWAJqvdrs8a -k -d "grant_type=password&username=larry&password=larry123&scope=uma_protection internal_application_mgt_view" -H "Content-Type:application/x-www-form-urlencoded" https://localhost:9443/oauth2/token
+    ```
+
+-   You will get a response similar to the following:
+
+    ```
+    {
+        "access_token":"b8df48ff-feab-3632-b3dc-68ae6b4c62e2",
+        "refresh_token":"1037ccad-f45a-38e7-96ad-40c00fbc7ca4",
+        "scope":"internal_application_mgt_view uma_protection",
+        "token_type":"Bearer",
+        "expires_in":3600
+    }
+    ```
+
+---
+
+## Register the resource
+
+Now, you need to register the resource.
+
+-   Execute the following curl command to put the resouce owner's resource under authorization server (WSO2 IS) protection.
+    -   Make sure to replace the `<PAT>` tag with the [access token you got in the previous section](#obtain-the-protection-api-access-token-pat).
```

Comment:
```suggestion
    -   Be sure to replace the `<PAT>` tag with the [access token you got in the previous section](#obtain-the-protection-api-access-token-pat).
```

Reply:
Can we make this a note?

Reply:
Is it possible to add notes to a github doc?